### ISSUE TITLE
Pycortex not showing

### ISSUE THREAD

**Issue by eglerean** [NeuroVault/NeuroVault#704]
Hello!

When teaching the tool to students, I have noticed that some recent maps are not showing the pycortex visualization and return a 500 Server Error.

E.g. https://neurovault.org/collections/5044/

thank you!!
Enrico

**Comment by Shotgunosine** [comment]
Yep, getting 500 errors on collections [https://neurovault.org/collections/4022/](https://neurovault.org/collections/4022/)and [https://neurovault.org/collections/4022/](https://neurovault.org/collections/4978/)
